doc/classes/Dictionary.xml

```diff
@@ -87,6 +87,14 @@
 				Return the list of values in the [code]Dictionary[/code].
 			</description>
 		</method>
+		<method name="get">
+			<return type="Variant">
+			</return>
+			<description>
```

Comment:
The argument seems to be missing:
```
<argument index="0" name="default" type="Variant">
</argument>
```

Reply:
@akien-mga Thanks for the suggestion.  I just updated the PR to add the missing arguments in the .get() method documentation.